frontend/src/types.ts

```diff
@@ -375,6 +375,7 @@ export interface OrganizationDomainType {
 export interface BaseMemberType {
     id: string
     user: UserBasicType
+    last_login?: string
```

Comment:
Looks like this will always be present, not optionally – even though it may be null:
```suggestion
    last_login: string | null
```